R/uptake.R

```diff
@@ -16,11 +16,11 @@ get_scenario_go_dutch = function(routes, purpose = "work") {
         bicycle_go_dutch = pcycle_go_dutch * all,
         bicycle_increase = bicycle_go_dutch - bicycle,
         car_driver_go_dutch = car_driver - (bicycle_increase * car_driver / all),
-        car_passenger_go_dutch = car_passenger - (bicycle_increase * car_passenger / all),
+        #car_passenger_go_dutch = car_passenger - (bicycle_increase * car_passenger / all),
```

Comment:
We don't need car for now. How hard would it be to add @mem48 ?

Reply:
In Scotland there is no driver/passenger  split but we do have car as a single category